kafka-ui-react-app/src/components/Topics/Topic/Details/__test__/Details.spec.tsx

```diff
@@ -44,7 +44,7 @@ describe('Details', () => {
 
   describe('when it has readonly flag', () => {
     it('does not render the Action button a Topic', () => {
-      const component = mount(
+      const component = render(
         <ThemeProvider theme={theme}>
```

Comment:
`render` from `lib/testHelpers` already has `<ThemeProvider>` (you can look at implementation) 
I think you can safely remove it here and other places (i won't spam with them, just search for *.spec.tsx files with `ThemeProvider`)